src/gui/widgets/registers.cc

```diff
@@ -40,8 +40,10 @@ void PCSX::Widgets::Registers::draw(PCSX::GUI* gui, PCSX::psxRegisters* register
     if (ImGui::BeginTabBar(_("Registers"))) {
         gui->useMonoFont();
         if (ImGui::BeginTabItem("GPR")) {
+            std::vector<std::pair<const char*, uint32_t>> regs;
```

Comment:
Let's avoid this pattern - it'll be run on every refresh still... instead let's just add a `ImGui::Separator()` at the end of the loop and add `pc` there in a special manner. We should also probably just factor all the "show a register" code into a separate function to be able to reuse it on any register, not just the GPRs.